templates/master.yaml.tmpl

```diff
@@ -1423,7 +1423,7 @@ storage:
                   command:
                   - /hyperkube
                   - proxy
-                  - --proxy-mode=ipvs
+                  - --proxy-mode=iptables
```

Comment:
Is it possible to remove this flag ? and therefore only having the proxy-mode configured in one place (`kube-proxy.yaml`).

Reply:
I guess this will be temporary until ipvs gets stable, I guess makes sense to keep that in order to make easier future change to ipvs.